parsl/executors/high_throughput/executor.py

```diff
@@ -93,6 +94,14 @@ class HighThroughputExecutor(ParslExecutor, RepresentationMixin):
     worker_port_range : (int, int)
         Worker ports will be chosen between the two integers provided.
 
+    interchange_address : str | function
+        Address (str) at which the interchange will be reachable from compute nodes or an address
+        function from parsl.addresses that be executed on the interchange to determine a suitable
+        address. Only valid when remote_interchange=True.
```

Comment:
Description might need to be updated as we decide to remove `remote_interchange=True`